modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/mv/PersistentPageMemoryMvPartitionStorage.java

```diff
@@ -136,7 +130,9 @@ public CompletableFuture<Void> flush() {
         if (lastCheckpoint != null && meta.metaSnapshot(lastCheckpoint.id()).lastAppliedIndex() == meta.lastAppliedIndex()) {
             scheduledCheckpoint = lastCheckpoint;
         } else {
-            PersistentPageMemoryStorageEngineView engineCfg = tableStorage.engine().configuration().value();
+            var persistentTableStorage = (PersistentPageMemoryTableStorage) tableStorage;
```

Comment:
is `var` ok here because of the cast?

Reply:
Well, formally it's not, but come on, the type is right there